analysers/analyser_osmosis_highway_tunnel_bridge.py

```diff
@@ -197,14 +205,15 @@ def __init__(self, config, logger = None):
         self.callback40 = lambda res: {"class": (4 if res[4] == 'bridge' else 5) + (2 if res[5] else 0), "data": [self.node_full, self.way_full, self.way_full, self.positionAsText] }
 
     def analyser_osmosis_full(self):
-        self.run(sql10.format(""), self.callback10)
+        self.run(sql10.format("", ""), self.callback10)
         #self.run(sql20.format("", ""))
         #self.run(sql21, self.callback20)
         #self.run(sql30.format("", ""), self.callback30)
         self.run(sql40.format("", ""), self.callback40)
 
     def analyser_osmosis_diff(self):
-        self.run(sql10.format("touched_"), self.callback10)
+        self.run(sql10.format("touched_", ""), self.callback10)
+        self.run(sql10.format("not_touched_", "touched_"), self.callback10)
```

Comment:
We cannot do this with a LEFT JOIN... IS NULL. `{1}` could not be restricted.

Reply:
Oh right... And because the man_made can be added without modifying the highway, the `{0}`  should also go